lib/logthrdest/logthrdestdrv.c

```diff
@@ -1034,13 +1043,41 @@ log_threaded_dest_driver_set_max_retries_on_error(LogDriver *s, gint max_retries
   self->retries_on_error_max = max_retries;
 }
 
+static guint
+_get_worker_key_hash(LogThreadedDestDriver *self, LogMessage *msg)
+{
+  if (log_template_is_literal_string(self->worker_partition_key))
+    return g_str_hash(log_template_get_literal_value(self->worker_partition_key, NULL));
```

Comment:
Should we let the user set a literal string for this at all? I think that would be a clear misconfiguration.

Reply:
Nice catch. Fixed.